root/inc/module-install.html

```diff
@@ -0,0 +1,35 @@
+
+<script type="text/javascript">
```

Comment:
I was going to suggest that we don't use inline scripts and to put this logic into `./root/static/js/cpan.js`, but I see that we do have other templates with `<script>` tags.  I would think moving it to cpan.js would be the best way to handle this. @rwstauner, do you have an opinion?